jenkins/common.groovy

```diff
@@ -131,6 +131,39 @@ def setup() {
   }
 }
 
+def checkForBrokenLinks(String buildLogPath, String envName) {
+  // Look for broken links
+  def csvFileName = "${envName}-broken-links.csv" // For use within the docker container
+  def csvFile = "${WORKSPACE}/vets-website/${csvFileName}" // For use outside of the docker context
```

Comment:
Ah, I wonder if the `${WORKSPACE}` here is why we weren't finding the build output before.

Reply:
Yeah, exactly.